docs/guides/getting_started.md

```diff
@@ -408,10 +408,30 @@ const links = $('a[href]')
     .map((i, el) => $(el).attr('href'))
     .get();
 
-const ourDomain = 'apify.com';
-const absoluteUrls = links.map(link => new URL(link, ourDomain));
+const {protocol, hostname} = new URL(request.loadedUrl);
+const absoluteUrls = links.map(link => new URL(link, request.loadedUrl));
+
+const sameDomainLinks = absoluteUrls.filter(url => url.protocol === protocol && url.hostname.endsWith(hostname));
+
+// ...
+```
+
+This includes subdomains. In order to pin the entire origin, simply compare the `url.origin` property instead:
+
+```js
+// At the top of the file:
+const { URL } = require('url');
+
+// ...
+
+const links = $('a[href]')
+    .map((i, el) => $(el).attr('href'))
+    .get();
+
+const {origin} = new URL(request.loadedUrl);
+const absoluteUrls = links.map(link => new URL(link, request.loadedUrl));
 
-const sameDomainLinks = absoluteUrls.filter(url => url.href.startsWith(ourDomain));
+const sameDomainLinks = absoluteUrls.filter(url => url.origin === origin);
```

Comment:
I don't think this second example is needed in full length. The two changed lines should be enough.